src/interfaces/controller/notification/notification.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Controller, UseGuards, Get, Req, Res, HttpStatus } from '@nestjs/common';
+import { Request, Response } from 'express';
+import { JwtAuthGuard } from 'src/interfaces/controller/common/guards/jwt-auth.guard';
+import { NotificationUseCase } from 'src/application/notification/notification.use-case';
+import { NotificationsResponseDto } from './dto/notification.response.dto';
+import { NotificationsRequestDto } from './dto/notification.request.dto';
+import { ResultResponseDto } from 'src/interfaces/controller/common/dto/result.response.dto';
+
+@UseGuards(JwtAuthGuard)
+@Controller('notifications')
+export class NotificationController {
+  constructor(private readonly notificationUseCase: NotificationUseCase) {}
+
+  @Get('')
+  async getNotification(@Req() req: Request, @Res() res: Response) {
+    const query = new NotificationsRequestDto(req);
+    const result = await this.notificationUseCase.getNotification(query);
```

Comment:
```suggestion
  async getNotifications(@Req() req: Request, @Res() res: Response) {
    const query = new NotificationsRequestDto(req);
    const result = await this.notificationUseCase.getNotifications(query);
```
사소한 디테일이긴 하지만, 여러개의 복수의 값을 가져오는 것이니 s를 붙여주는게 어떨까요???